README.txt

```diff
@@ -170,6 +170,7 @@ ENABLE_FREETYPE     - Build with FreeType2; Allows using TTF fonts
 ENABLE_GETTEXT      - Build with Gettext; Allows using translations
 ENABLE_GLES         - Search for Open GLES headers & libraries and use them
 ENABLE_LEVELDB      - Build with LevelDB; Enables use of LevelDB map backend (faster than SQLite3)
+ENABLE_POSTGRESQL   - Build with libpq; Enables use of PostgreSQL map backend (faster than LevelDB and less memory consuming than Redis)
```

Comment:
Maybe the virtues of each DB should be explained in a separate section of the README, not inline with each build flag.